source/jupyter.rst

```diff
@@ -0,0 +1,148 @@
+.. jupyter:
+
+Installing Jupyter
+------------------
+
+Please see project Jupyter's `instructions for installation
+<http://jupyter.readthedocs.io/en/latest/install.html>`__. Basically,
+
+-  **If you already have Python (2 or 3)**, follow the following
+   instructions for your particular OS from the above webpage. We will
+   use Python 3 henceforth.
+
+-  **If you are a new Python user, or just want to reinstall Python
+   (say, to move from Python 2 to 3)** you can install Python+Jupyter
+   using `Anaconda <https://www.continuum.io/downloads>`__, by
+   downloading Anaconda's latest Python 3 version.
+
+Ubuntu / Linux
+~~~~~~~~~~~~~~
+
+As an existing Python 3 user, you may wish to install Jupyter using
+Python's package manager, ``pip``, instead of Anaconda. Do the
+following:
+
+.. code:: console
+
+   sudo pip3 install --upgrade pip
+
+   pip3 install jupyter
+
+If pip3 is not available, then first:
+
+.. code:: console
+
+   sudo apt-get install python3-pip
+
+If you keep getting a ``pip2 command not found`` error:
+
+.. code:: console
+
+   sudo apt remove python3-pip && sudo apt install python3-pip
+
+..
+
+   If you already have python 2.x and then installed python3, your pip
+   will be pointing to pip3. you can verify that by typing
+   ``pip --version`` which would be the same as ``pip3 --version``.
```

Comment:
A lot of this mucking around with which pip version is in use is avoided by just specifying the Python interpreter to use. I.e.:

``` bash
python3 -m pip install Jupiter
```